docs/management-ui.mdx

```diff
@@ -65,3 +65,35 @@ curl -X POST "https://api.svix.com/api/v1/auth/dashboard_access/app_Xzx8bQeOB1D1
 
 </TabItem>
 </CodeTabs>
+
+
+## Embedding in your own dashboard
+
+The returned URL from the previous section can also be embedded in your own dashboard using an iframe. This is how it looks like:
+
+<img alt="Embedded Management UI screenshot" src={useBaseUrl("img/embedded-management-ui.png")} />
+
+To do it, just set the url from the examples above in the `src` property of the iframe:
+
+```html
+<iframe
+    src="http://localhost:3000/login#key=eyJhcHBJZCI6ICJhcHBfMXRSdFlMN3pwWWR2NHFuWTRRZFI1azE4eXQ0IiwgInRva2VuIjogImFwcHNrX2UxOUN0Rm5hbTFoOU1Gamh5azRSMTUzNUNSd05VSWI0In0="
+>
+</iframe>
+```
+
+Or with some basic styling to make it look better:
+
+```html
+<iframe
+    src="http://localhost:3000/login#key=eyJhcHBJZCI6ICJhcHBfMXRSdFlMN3pwWWR2NHFuWTRRZFI1azE4eXQ0IiwgInRva2VuIjogImFwcHNrX2UxOUN0Rm5hbTFoOU1Gamh5azRSMTUzNUNSd05VSWI0In0="
+    style="width: 100%; height: 100%; border: none;"
+>
+</iframe>
+```
+
+## Customizing the look and feel
+
+The management UI automatically can be customized to follow your UI's look and feel. We currently support customizing the color palette as well as the logo used.
+
+To customize these settings please head to [the dashboard](https://dashboard.svix.com) and change it under "Settings".
```

Comment:
```suggestion
To customize these settings, please head to your [Organization Settings](https://dashboard.svix.com/settings) on the Svix Dashboard.

XXX Add screenshot here
```

Reply:
I removed the XXX to add a screenshot. We can do it in a later PR.

Reply:
I did it because I thought I could just merge it, but I guess I need to wait for the dashboard changes to be deployed first. So I'm reopening this comment. Please share a screenshot once you have one.